### ISSUE TITLE
Ableton 10 unresponsive after adding 2nd instance

### ISSUE THREAD

**Issue by TrySpace** [holzon/the-seeker#1]
On windows 10, latest ableton 10.

To reproduce, add it to one track, then go to another track and ad it.

The user interface will become unresponsive, although any external midi controllers will still work, and visually everything still animates, including the plugin. You can still switch between tracks etc. if you have a controller for that.

It's as if there's a layer above ableton that does not allow me to click/interact anymore.

Once ableton is minimized it won't maximize anymore either.

Basically the only thing I can do is right click close ableton and it will ask if I want to save the project, but I can't click save/cancel either..

It looks like the plugin is requesting exclusive focus
